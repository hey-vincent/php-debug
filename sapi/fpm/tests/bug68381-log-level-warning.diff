001+ ERROR: Expected no log lines but following lines logged:
002+ [24-Dec-2018 17:12:09] NOTICE: PHP message: PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/local/lib/php/extensions/no-debug-non-zts-20151012/iconv' - dlopen(/usr/local/lib/php/extensions/no-debug-non-zts-20151012/iconv, 9): image not found in Unknown on line 0
003+ 
004+ [24-Dec-2018 17:12:09] NOTICE: PHP message: PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/local/lib/php/extensions/no-debug-non-zts-20151012/openssl' - dlopen(/usr/local/lib/php/extensions/no-debug-non-zts-20151012/openssl, 9): image not found in Unknown on line 0
005+ 
006+ [24-Dec-2018 17:12:09] NOTICE: PHP message: PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/local/lib/php/extensions/no-debug-non-zts-20151012/zip' - dlopen(/usr/local/lib/php/extensions/no-debug-non-zts-20151012/zip, 9): image not found in Unknown on line 0
007+ 
008+ [24-Dec-2018 17:12:09] NOTICE: PHP message: PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/local/lib/php/extensions/no-debug-non-zts-20151012/mbstring' - dlopen(/usr/local/lib/php/extensions/no-debug-non-zts-20151012/mbstring, 9): image not found in Unknown on line 0
009+ 