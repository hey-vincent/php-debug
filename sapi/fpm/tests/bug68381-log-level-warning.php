<?php

require_once "tester.inc";

$cfg = <<<EOT
[global]
error_log = {{FILE:LOG}}
log_level = warning
[unconfined]
listen = {{ADDR}}
pm = dynamic
pm.max_children = 5
pm.start_servers = 2
pm.min_spare_servers = 1
pm.max_spare_servers = 3
EOT;

$tester = new FPM\Tester($cfg);
$tester->start();
$tester->checkConnection();
$tester->terminate();
$tester->expectNoLogMessages();
$tester->close();

?>
Done
