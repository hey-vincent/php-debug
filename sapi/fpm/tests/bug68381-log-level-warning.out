ERROR: Expected no log lines but following lines logged:
[24-Dec-2018 17:12:09] NOTICE: PHP message: PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/local/lib/php/extensions/no-debug-non-zts-20151012/iconv' - dlopen(/usr/local/lib/php/extensions/no-debug-non-zts-20151012/iconv, 9): image not found in Unknown on line 0

[24-Dec-2018 17:12:09] NOTICE: PHP message: PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/local/lib/php/extensions/no-debug-non-zts-20151012/openssl' - dlopen(/usr/local/lib/php/extensions/no-debug-non-zts-20151012/openssl, 9): image not found in Unknown on line 0

[24-Dec-2018 17:12:09] NOTICE: PHP message: PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/local/lib/php/extensions/no-debug-non-zts-20151012/zip' - dlopen(/usr/local/lib/php/extensions/no-debug-non-zts-20151012/zip, 9): image not found in Unknown on line 0

[24-Dec-2018 17:12:09] NOTICE: PHP message: PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/local/lib/php/extensions/no-debug-non-zts-20151012/mbstring' - dlopen(/usr/local/lib/php/extensions/no-debug-non-zts-20151012/mbstring, 9): image not found in Unknown on line 0

Done